Makefile

```diff
@@ -0,0 +1,12 @@
+help:
+	@echo "This project supports the following targets"
+	@echo ""
+	@echo " make help - show this text"
+	@echo " make build - build the snap"
+	@echo ""
+
+build:
+	@echo update snap hooks and local
+	@bash -c ./sync_upstream.sh
+	@echo "Building the snap"
+	@snapcraft --use-lxd
```

Comment:
I'm in favour of removing this Makefile.  I think the sync from upstream script is for convenience only if we ever want to resync from upstream; we don't want to run it every time we build the snap.

Better to simply document:

- `snapcraft` to build the snap
- `./sync_upstream.sh` to update the files from upstream if needed at some point